src/App.js

```diff
@@ -1,10 +1,9 @@
-/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
 import CalculatorBody from './components/calculatorBody';
 
-class App extends React.Component {
+class App extends React.PureComponent {
```

Comment:
- [ ] [OPTIONAL] You made great job in here! Making it PureComponent. However it would be even better if you could make it function component as well!

Reply:
Hi @Rustamxon7 
Nice suggestion.
Thanks,